tests/rbd_mirror/test_rbd_mirror_journal_to_snap.py

```diff
@@ -56,25 +56,13 @@ def test_journal_to_snapshot(rbd_mirror, pool_type, **kw):
     """
     try:
         log.info("Starting RBD mirroring test case")
-        config = kw.get("config")
+        config = kw.get("config", {}).get(pool_type)
         mirror1 = rbd_mirror.get("mirror1")
         mirror2 = rbd_mirror.get("mirror2")
-        poolname = mirror1.random_string() + "_tier_2_rbd_mirror_pool"
-        imagename = mirror1.random_string() + "_tier_2_rbd_mirror_image"
+        poolname = config.get("pool")
+        imagename = config.get("image")
         imagespec = poolname + "/" + imagename
 
-        # initial mirror configuration
```

Comment:
why we removed these lines?

Reply:
Because pool and image are already getting created when we call rbd_mirror_config method in run method. We don't need to again call initial_mirror_config to create pool and image again.